cmd/performance-profile-creator/cmd/root.go

```diff
@@ -30,15 +30,16 @@ import (
 
 	performancev2 "github.com/openshift-kni/performance-addon-operators/api/v2"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-const (
-	singleNumaNodeTopologyManagerPolicy = "single-numa-node"
+	kubeletconfig "k8s.io/kubelet/config/v1beta1"
 )
 
 var (
-	validTMPolicyValues        = []string{"single-numa-node", "best-effort", "restricted", "none"}
-	validPowerConsumptionModes = []string{"cstate", "no-state", "idle-poll"}
+	validTMPolicyValues = []string{kubeletconfig.SingleNumaNodeTopologyManager, kubeletconfig.BestEffortTopologyManagerPolicy, kubeletconfig.RestrictedTopologyManagerPolicy}
+	// Values part of validPowerConsumptionModes are explained below:
+	// default => Use CPU C-states but limit them to 1, meaning the processor is idle it can sleep but not "deep sleep"
+	// performance => Disable CPU sleep (c-states), processor never sleeps even if is idle
```

Comment:
intel_idle.max_cstate=0 processor.max_cstate=0 intel_pstate=disable

Reply:
- "processor.max_cstate=1" <- This does not support 0 internally, it converts it to 1 anyway
    - "intel_idle.max_cstate=0"